apps/desktop/src/lib/file/FileListItem.svelte

```diff
@@ -109,10 +116,11 @@
 	}
 
 	onDestroy(() => {
+		chips?.destroy();
 		if (draggableEl && animationEndHandler) {
 			draggableEl.removeEventListener('animationend', animationEndHandler);
 		}
-		draggableFiles.forEach((f) => {
+		$selectedFiles.forEach((f) => {
 			const lockedElement = document.getElementById(`file-${f.id}`);
 			if (lockedElement && animationEndHandler) {
 				lockedElement.removeEventListener('animationend', animationEndHandler);
```

Comment:
This doesn't guarantee that we remove all the event listeners. We ought to record all the elements that have the event added to them, and then iterate through that set and remove the event listeners added to those elements